### PR TITLE
Fallback to UUID on O.E.M string

### LINKED ISSUE #7: weird serrial
On one hp laptop i got this for seriall number:
`To be filled by O.E.M.` . Do you have any idea why is this not falling at least to UUID ?

### PR DESCRIPTION
**Pull request by dcrystalj**
trying to fix https://github.com/es128/serial-number/issues/7

### CODE REVIEW

**Comment by dcrystalj** [comment]
are there any other known useless strings?

**Comment by es128** [comment]
At some point I've seen another useless value, but don't quite remember what it was. The solution in that particular implementation was just to hardcode in the `UUID` option.

So assuming there are others, would you mind refactoring this as an array using something like `[].some()` so that it will be easy to add new entries in the future?

**Comment by dcrystalj** [comment]
not sure what do you mean by [].some() . please check refactored code

**Comment by es128** [comment]
The `in` keyword is for keys in an object, not values in an array. That solution won't work.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes would be ideal, but it's a newer javascript feature and I want to preserve compatibility with older versions of node, so I suggested doing it with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

**Comment by dcrystalj** [comment]
sorry for inconvenience.

File: index.js [hunk 1]
```diff
@@ -29,7 +32,17 @@ var serialNumber = function (cb, cmdPrefix) {
 	};
 
 	var parseResult = function (input) {
-		return input.slice(input.indexOf(delimiter) + 2).trim();
+		result = input.slice(input.indexOf(delimiter) + 2).trim();
```
Comment (es128) [inline index.js]: `var result`

**Comment by es128** [comment]
No inconvenience, and no apology necessary. 

Thank you for putting together the PR and responding to feedback.

**Comment by dcrystalj** [comment]
I'm glad to learn something new :+1:

**Pull request merged**